website/docs/reference/activation-strategies.md

```diff
@@ -2,130 +2,167 @@
+## Overview 
 
-All [server-side client SDKs](../reference/sdks#server-side-sdks) and the [Unleash Proxy](../reference/unleash-proxy) implement the default strategies (and allow you to add your own [custom strategy implementations](../reference/custom-activation-strategies#implementation)). The [front-end client SDKs](../reference/sdks#front-end-sdks) do not do the evaluation themselves, instead relying on the [Unleash Proxy](../reference/unleash-proxy) to take care of the implementation and evaluation.
+An activation strategy determines who should get a feature. They allow you to enable and disable features for certain users without having to redeploy your application. For example, you can release features only to users with a specific user ID, email, IP address, and more. You can implement gradual rollouts to specific user segments, for example, those on a specific subscription plan or region. You can also use them to schedule feature releases or make features available for a limited time.
 
-Some activation strategies require the client to provide the current [Unleash context](unleash-context) to the flag evaluation function for the evaluation to be done correctly.
+An activation strategy is assigned to one [feature flag](/reference/feature-toggles) in one [environment](/reference/environments). A feature flag is enabled in a given context (for example, user or application) if at least one of its activation strategies resolves to true.
 
-The following activation strategies are bundled with Unleash and always available:
+You can copy activation strategies from one environment to the other, but the different strategy configurations do not stay in sync. The default activation strategy is a 100% gradual rollout, enabling the flag for all users. You can refine this using [rollout percentage](#rollout-percentage), [targeting](#targeting), and [variants](/reference/strategy-variants).
 
-- [Standard](#standard)
-- [UserIDs](#userids)
-- [Gradual Rollout](#gradual-rollout)
-- [IPs](#ips)
-- [Hostnames](#hostnames)
+Feature flags can have multiple activation strategies. Unleash evaluates each strategy independently, enabling the flag if any resolves to true. This behavior is equivalent to the OR logical operator.
 
-## Standard {#standard}
+For example, to roll out a feature to 75% of users while granting access to internal users, you can add two activation strategies as follows: 
+1. Gradual rollout to 75%.
+2. Gradual rollout to 100% with a constraint on the email address.
```

Comment:
```suggestion
1. Gradual rollout to 100% with a constraint on the email address (e.g. email ends with @mydomain.com).
2. Gradual rollout to 75% (for the remaining).
```
In this particular case order doesn't matter because if your email ends with @mydomain.com you can either be enabled because you also fall into the 75% or because your email matches, but (personally) I think it reads better if the more restrictive rule is on top

Reply:
Discussed offline, order doesn't matter, OR is commutative